lnwallet/chainfee/estimator.go

```diff
@@ -306,9 +311,12 @@ func NewBitcoindEstimator(rpcConfig rpcclient.ConnConfig,
 		return nil, err
 	}
 
+	log.Debugf("bitcoind's fee estimate mode is set to: %s", feeMode)
```

Comment:
Nit: this log isn't necessary since we already log it in `chainregistry.go`.

Reply:
fixed.